app/routes/contracted-hours.js

```diff
@@ -0,0 +1,42 @@
+const contractedHoursService = require('../services/contracted-hours-service')
+const getSubNav = require('../services/get-sub-nav')
+const organisationUnitConstants = require('../constants/organisation-unit')
+
+module.exports = function (router) {
+  router.get('/:organisationLevel/:id/contracted-hours', function (req, res, next) {
+    var organisationLevel = req.params.organisationLevel
+    var id = req.params.id
+
+    if (organisationLevel !== organisationUnitConstants.OFFENDER_MANAGER.name) {
+      return res.sendStatus(404)
```

Comment:
What was the thought behind changing to a `404`?

Reply:
Usually we would return a 500, but that should be reserved for actual internal errors.  Thought I should return 404 not found because that url shouldn't exist for anything other than offender-manager.  Alternative could be to hard-code offender-manager into the get url?

Reply:
Ok with this 404 in theory, but would like all our routes to be consistent, one way or the other.